wandb/sdk/internal/internal_api.py

```diff
@@ -2454,7 +2454,23 @@ def commit_artifact(self, artifact_id):
         }
         """
         )
-        response = self.gql(mutation, variable_values={"artifactID": artifact_id})
+
+        def check_retry(e: Any) -> bool:
+            if util.no_retry_auth(e):
+                return True
+            if hasattr(e, "exception"):
+                e = e.exception
+            if isinstance(e, requests.HTTPError) and e.response.status_code == 409:
+                logger.error("conflict when trying to commit artifact; retrying")
```

Comment:
This will add _a lot_ of entries to our logs + a bunch of tests now will prob fail (there are potentially more entries in logs than some tests expect? not 100% sure tho) ;)

Reply:
Fair 'nuff! Removed!